src/Web/Areas/Admin/Views/Shared/_ValidationScriptsPartial.cshtml

```diff
@@ -1,23 +1,13 @@
 @*
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.   
+    // Copyright (c) Microsoft Corporation.
+    // Licensed under the MIT license.
 *@
 
-<environment names="Development">
-    <script src="~/lib/jquery-validation/dist/jquery.validate.js"></script>
```

Comment:
I see you're removing the server-hosted script tags; I don't see the files being deleted in this PR though?

Reply:
Haha!  And that's why they're being removed... they don't exist any longer.  I imagine they did at one point and got removed, but these lines did not.  Furthermore, it was extra complexity for no benefit in a sample app.